### DISCUSSION TITLE
Ability for a room admin to shutdown a room.

### DISCUSSION THREAD

**Discussion by ara4n** [element-hq/element-meta#919] [Ideas]
Whilst server admins can shutdown rooms, it'd be useful for a room admin to be able to close a room and repoint it to another one (as a form of merging).

**Comment by lampholder** [comment]
If I understand the ask correctly, I think this can be sort-of achieved by the admin:
- removing admin perms from all other users
- raising the powerlevel required to speak to admin
- saying the last thing in the room: 'this room is now closed - go to #greatnewroom:matrix.org'
- perhaps redirecting any aliases to the new room

(Only referencing how it can be achieved for people who find this in search results and to back up my prioritisation).

@ara4n - what would this look like as a proper feature? Would navigating to the actual room id still redirect you to the new room, etc?

**Comment by Sancus** [comment]
It's a fairly common(and very useful!) workflow in Slack to create a small room for a project temporarily, and then shutdown+archive it once that project is done.

So it'd be nice if there was UI to tombstone a channel similarly to this, and it'd be even better if that channel went into an archive or after leaving it it went into an archive that the individual user could keep or delete(up to them).
